.flake8

```diff
@@ -1,4 +1,6 @@
 [flake8]
 # Ignore some errors, since we autoformat them away already wherever possible
 # from https://github.com/psf/black/blob/main/.flake8
-ignore = E203, E266, E501, W503
+# ignore = E203, E266, E501, W503
+ignore = D107, W503, E203  # should align black and flake8 'rules'
+max-line-length = 120
```

Comment:
Thanks for noting this, I dislike black so much for imposing by default such short lines that make the code unnecessarily long! Once I tried changing this parameter in the etl and it did not work, I hope here it works.